www/src/components/languages/language-table-row.js

```diff
@@ -0,0 +1,96 @@
+/** @jsx jsx */
+import { jsx } from "theme-ui"
+import styled from "@emotion/styled"
+import { css } from "@emotion/core"
+import { mediaQueries } from "gatsby-design-tokens/dist/theme-gatsbyjs-org"
+import { OutboundLink } from "gatsby-plugin-google-analytics"
```

Comment:
let's just use a normal `<a>` tag here. The `OutboundLink` is specifically a link that will track clicks on google analytics. We're currently using it only for the banner to track who goes to check out Gatsby Cloud.

Reply:
Ah, understood!